trace.py

```diff
@@ -15,12 +15,13 @@ def __init__(self, filename: str):
     self._filename = filename
 
   def _get_direction_filter(self, d: Direction) -> str:
+    f = "(quic && !icmp) && "
```

Comment:
```suggestion
    f = "(quic) && "
```
Is this not enough?

Reply:
For reasons I don't fully understand it's not.